src/handlers/crudHandler.js

```diff
@@ -61,13 +61,13 @@ function deleteHandler(collectionName, req, res) {
 
   // Also delete subscription from EHR
   if (collectionName === SUBSCRIPTIONS) {
-    deleteSubscriptionFromEHR(resource);
+    deleteSubscriptionFromEHR(resource.resource);
```

Comment:
Maybe the variable name shouldn't be resource if it's not actually a FHIR resource. I'd propose `entry` or `resourceWrapper` or just `deletedItem` because that's how it's being used, a link back to the item that was deleted.

Reply:
renamed to `entryToDelete`